nz/anz/README.md

```diff
@@ -0,0 +1,15 @@
+# ANZ config files
+These config files provide configuration for exports from ANZ New Zealand accounts.
+
+## Which file?
+The file "default.json" is the standard config for most accounts (e.g. Go, Flexiloan).
```

Comment:
If you want a new paragraph, be sure to add a double enter.

Reply:
Cheers, have updated